templates/chats/chat_group.html

```diff
@@ -48,7 +48,7 @@
         </div>
         <div class="sticky bottom-0 z-10 p-2 bg-gray-800">
             <div class="flex items-center rounded-xl px-2 py-2">
-                <form id="chat_message_form" class="w-full" method='post'>
+                <form action="" id="chat_message_form" class="w-full" method='post'>
```

Comment:
`action` 是空的有什麼特別原因嗎？ 
```suggestion
                <form action="" id="chat_message_form" class="w-full" method='post'>
```

Reply:
是要讓表單送至自己，那時候在測試忘記刪掉了